### ISSUE TITLE
[2.4.4] Cann't build apk cocos creator

### ISSUE THREAD

**Issue by deploygo88** [cocos/cocos-engine#14604]
### Cocos Creator version

2.4.4

### System information

Windows 10 Pro

### Issue description

Cocos creator: 2.4.4
NDK: 19.2.5345600
JDK: jdk-8u361
Python: 2.7.18

I cann't build apk even though i create example hello word project from cocos creator with 2.4.4, but when i build 2.2.2 it's success
How can i build with 2.4.4? My project current using this version, thanks for reading

### Steps to reproduce

This is my detail error when i build with cocos creator 2.4.4
[native.log](https://github.com/cocos/cocos-engine/files/11012612/native.log)

**Comment by minggo** [comment]
@PatriceJiang please take a look.

**Comment by PatriceJiang** [comment]
To obtain detailed logs, you can cd to the root directory of the Android project and execute ./gradlew --debug assembleDebug. @deploygo88

**Comment by deploygo88** [comment]
![image](https://user-images.githubusercontent.com/126611757/226337314-823c3cd2-35c4-48da-b146-27903b0bda54.png)
![image](https://user-images.githubusercontent.com/126611757/226339594-ae63ac1a-2359-456f-bd90-ffae87143e73.png)
![image](https://user-images.githubusercontent.com/126611757/226339630-4bbfc44b-aac0-45d4-ba23-716f144b8f02.png)
![image](https://user-images.githubusercontent.com/126611757/226339668-5be49f5b-5379-4b24-a04c-06aa9ee5b391.png)

I don't know if this is what you want or not. Here are the logs after I run the command ./gradlew --debug assembleDebug @PatriceJiang

**Comment by PatriceJiang** [comment]
Based on the log content, it seems the build was successful. 

If there are still errors, you can try opening the project in Android Studio and attempt to compile it there. 

It's difficult to make out the details from the screenshot provided.

**Comment by deploygo88** [comment]
I still have errors when using this version, I wanted to know , which version of ndk version, Android Gradle plugin version and Gradle version that needs to be used. Thanks @PatriceJiang

**Comment by PatriceJiang** [comment]
Please refer to the documentation at https://docs.cocos.com/creator/2.4/manual/en/publish/setup-native-development.html#android-gradle-upgrade

**Comment by deploygo88** [comment]
Yes,, I've read it before i built, but it not work (Windows 10 pro). When my friends test build with macOS. It built success. @PatriceJiang
